scripts/screens/base_screens.py

```diff
@@ -169,6 +169,7 @@ def menu_button_pressed(self, event):
         elif event.ui_element == self.menu_buttons["camp_screen"]:
             self.change_screen('camp screen')
         elif event.ui_element == self.menu_buttons["catlist_screen"]:
+            Cat.sort_cats()
```

Comment:
We don't want to be resorting the entire cat list whenever we open the catlist screen. This has the potential to cause lag on very large clans, and is a good amount of unneeded duplicate work. 

The bug with medicine cat may be caused be small differences in the behavior of `list.sort` and bisect.insort`.  For now, I'm not sure how to fix this issue, and it might be more of a pain to fix then it's worth.